examples/vue-webpack/src/App.vue

```diff
@@ -15,3 +20,30 @@ export default {
   },
 }
 </script>
+
+<style>
+.Example-List {
```

Comment:
These styles are duplicated, doesn't look good

Reply:
Right, needs to be combined to keep all examples the same